glue/qt/widget_properties.py

```diff
@@ -237,9 +247,15 @@ def connect_current_combo(client, prop, widget):
     """
 
     def _push_combo(value):
-        idx = widget.findData(value)
-        if idx == -1:
-            return
+        # NOTE: we can't use findData here because if the data is not a 
+        # string, PySide will crash
```

Comment:
(well maybe, I'm not sure how much metaclass magic happens when you instantiate a QWidget, and whether that makes overriding methods tricky)